CHANGELOG.yaml

```
@@ -11,6 +11,14 @@ meta:
     - clients: the service API changed in a way that requires clients to be updated
 
 changes:
+  - title: CloudWatch resource metrics enabled for non personal deployments with Terraform
+    issues:
+      - https://github.com/DataBiosphere/azul/issues/332
+    upgrade:
+      - terraform
+    notes: Personal deployments should not be affected by this change. This is only for the main deployments to achieve
```

Too much reasoning and technical detail. I'd remove all but the first sentence. 

And also:

```suggestion
    notes: Personal deployments are not affected by this change. This is only for the main deployments to achieve
```